create/how_to/updating.md

```diff
@@ -114,6 +114,10 @@ In this guide, we will:
 
 ### Step 1: Set all fields as displayed attributes
 
+::: note
+This step is not mandatory when creating dumps in MeiliSearch v0.21 and above.
```

Comment:
More than "not mandatory", it's completely useless and non-productive for the users. I would say "not necessary" or a synonym :)

Reply:
I'll even Write, `if your dump was created in MeiliSearch v0.21 or above, continue to step 2` or something similar

Reply:
I like the Caro's suggestion

Reply:
```suggestion
If your dump was created in MeiliSearch v0.21 or or above, continue to step 2.
```